keps/1136-provisioning-request-support/README.md

```diff
@@ -193,6 +193,33 @@ type ProvisioningRequestConfigSpec struct {
 	// +optional
 	// +kubebuilder:default={backoffLimitCount:3,backoffBaseSeconds:60,backoffMaxSeconds:1800}
 	RetryStrategy *ProvisioningRequestRetryStrategy `json:"retryStrategy,omitempty"`
+
+	// podSetUpdates specifies the update of the workload's PodSetUpdates which
+	// are used to target the provisioned nodes.
+	//
+	// +optional
+	PodSetUpdates *ProvisioningRequestPodSetUpdates `json:"podSetUpdates,omitempty"`
+}
+
+type ProvisioningRequestPodSetUpdates struct {
+	// nodeSelector specifies the list of updates for the NodeSelector.
+	//
+	// +optional
+	NodeSelector []ProvisioningRequestPodSetUpdatesNodeSelector `json:"nodeSelector,omitempty"`
```

Comment:
```suggestion
	// nodeSelectors specifies the list of updates for the NodeSelector.
	//
	// +optional
	NodeSelectors []ProvisioningRequestPodSetUpdatesNodeSelector `json:"nodeSelectors,omitempty"`
```

This can be accommodated the multiple nodeSelector"s".

Reply:
singluar `nodeSelector` is already a map of kep / value labels in k8s https://github.com/kubernetes/kubernetes/blob/1b4c5b3f8605dc1c86050ae941a24556d4fbd678/pkg/apis/core/types.go#L3539, so I would say it makes sense to say we construct single node selector, even though it make have multiple values, wdyt?

Reply:
The core API nodeSelector is a single selector to select candidate nodes.
Is the ProvReq nodeSelector the same functionality?

If yes, +1 on a singular `nodeSelector`.

Reply:
This is specification how to build the single node selector, yes. `[]ProvisioningRequestPodSetUpdatesNodeSelector ` is a list, where every element is responsible for determining one key-value pair. Typically this will be one element as showed in the example here: https://github.com/kubernetes-sigs/kueue/pull/5078#discussion_r2081970416

Reply:
Thank you for the clarification. SGTM